Makefile

```diff
@@ -24,7 +27,8 @@ test: build docs
 	python setup.py test pep8 pyflakes
 
 integration: build
-	PYFLAKES_NODOCTEST=True python setup.py integration pep8 pyflakes
+	INTEGRATION_TESTS=$(INTEGRATION_TESTS) PYFLAKES_NODOCTEST=True \
+		python setup.py integration pep8 pyflakes
```

Comment:
That works -- I agree -- but I don't actually think its simpler for most people. This setup allows people to run entire suites of tests with a simple command, rather than having to find each of the integration test python files.